externals/service_ittnotify.h

```diff
@@ -96,10 +96,19 @@ class ScopedTask
     #define DAAL_ITTNOTIFY_SCOPED_TASK(name)                                            \
         static daal::internal::ittnotify::StringHandle __ittnotify_stringhandle(#name); \
         daal::internal::ittnotify::ScopedTask __ittnotify_task(__ittnotify_domain, __ittnotify_stringhandle)
+
 #else
+    #include "service_profiler.h"
+
+    #undef DAAL_ITTNOTIFY_CONCAT
+    #define DAAL_ITTNOTIFY_CONCAT(x, y)  DAAL_ITTNOTIFY_CONCAT2(x, y)
+    #define DAAL_ITTNOTIFY_CONCAT2(x, y) x##y
+
+    #define __DAAL_UNIQUE_ID __LINE__
```

Comment:
Use consistent naming, if macro for concatenation named `DAAL_ITTNOTIFY_CONCAT2`, this one should be named `DAAL_ITTNOTIFY_UNIQUE_ID`

Reply:
Done